### PR TITLE
fix: replace uuid package with native crypto

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [X] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
Replace `uuid` package with native Node.js `crypto` package. Less 3rd party packages, less vulns.

### CODE REVIEW

**Review by chen-keinan — APPROVED** [review]

File: .circleci/config/jobs/@jobs.yml [hunk 1]
```diff
@@ -237,23 +229,18 @@ integration_tests_proxy:
...
   steps:
     - checkout
     - install_python_requests
+    - setup_node16
     - run:
         name: Create temp dir for logs
         command: mkdir -p /tmp/logs/test/integration/eks
     - run:
         name: Integration tests EKS
-        # WARNING! Do not use the step "setup_node16" here - the call to "nvm use 16" breaks the tests!
```
Comment (minsiyang) [inline .circleci/config/jobs/@jobs.yml:251]: Just wondering how did you pass this step now, as I can see that you have been using the ```setup_node16``` step now 🙂   And why was it breaking the tests before?

Comment (dkontorovskyy) [reply .circleci/config/jobs/@jobs.yml:251]: @minsiyang so I just decided to try it with `nvm use` and I don't have clue what is was breaking :) Tried searching in commit history. So just trying to figure :)

Comment (ivanstanev) [reply .circleci/config/jobs/@jobs.yml:251]: From what I remember it didn't interact well with python

**Review by ivanstanev — APPROVED** [review]

File: test/system/kind.spec.ts [hunk 2]
```diff
@@ -32,6 +33,9 @@ async function tearDown() {
   console.log('Begin removing the snyk-monitor...');
   try {
     await kind.deleteCluster();
+
+    // Workaround. Tests are failing, cos deleting cluster finishes after the test
+    await sleep(15 * 1000);
```
Comment (ivanstanev) [inline test/system/kind.spec.ts:38]: Huh?

Comment (dkontorovskyy) [reply test/system/kind.spec.ts:38]: 🤷‍♂️

https://app.circleci.com/pipelines/github/snyk/kubernetes-monitor/3433/workflows/358f0fe9-7136-4d94-b6e7-fdd4851c9560/jobs/18239

**Pull request merged**